### DISCUSSION TITLE
Nginx Proxy Manager Web Interface not loading anymore

### DISCUSSION THREAD

**Discussion by zerofruchtshake** [tteck/Proxmox#1024] [Help Needed]
**Describe the issue**
ERR_CONNECTION_TIMED_OUT when trying to access the web interface, was working before.
No recent changes to OS/config files
Proxies are still working

Where could i find relevant logs?

**Comment by tteck** [comment]
restore a backup?

**Reply by zerofruchtshake** [reply]
Sure I'll do that but i wanted to try to figure out the issue first. Do you know where i could find the logs?

**Reply by tteck** [reply]
have you looked in `/var/log`
